docs/installation/centos.md

```diff
@@ -53,7 +53,7 @@ package manager.
 
 3. Add the yum repo.
 
-        $ cat >/etc/yum.repos.d/docker.repo <<-EOF
+        $ echo | sudo tee /etc/yum.repos.d/docker.repo <<-EOF
```

Comment:
Why the `echo |`? Serves no purpose that I can see as STDIN to the `sudo tee` comes from the heredoc, not the `echo`.